### PR TITLE
TMA-484: Fix getting latest master version

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+module GoodData
+  module LCM2
+    class Helpers
+      DEFAULT_TABLE_NAME = 'LCM_RELEASE'
+
+      class << self
+        def latest_master_project(release_table_name, ads_client, segment_id)
+          replacements = {
+            table_name: release_table_name || DEFAULT_TABLE_NAME,
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Is controlled by argument 'release_table_name' (ControlParameter)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module LCM2
+    class Helpers
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has no descriptive comment (IrresponsibleModule)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      DEFAULT_TABLE_NAME = 'LCM_RELEASE'
+
+      class << self
+        def latest_master_project(release_table_name, ads_client, segment_id)
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has approx 7 statements (TooManyStatements)

File: lib/gooddata/lcm/helpers/release_table_helper.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module LCM2
```
Comment (yenkins) [inline lib/gooddata/lcm/helpers/release_table_helper.rb]: Has the name 'lcm2' (UncommunicativeModuleName)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/383/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1229/ : SUCCESS in 1m 05s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1499/ : SUCCESS in 2m 20s

**Pull request merged**